setup.py

```diff
@@ -10,7 +10,7 @@
     "pandas>=0.25.2",
     "mock>=4.0.2",
     "importlib-resources>=3.0.0",
-    "pathos>=0.2.7"
+    "pathos>=0.2.7",
 ]
```

Comment:
We can set a minimum version of python as well.
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires

Reply:
We do set a minimum as 3.6.8

Reply:
Because of library reasons, may want to update the version so do not need to manage incompatible versions.  Additionally PY3.7 is EOL. https://endoflife.date/python